django_tenants/middleware.py

```diff
@@ -26,7 +32,9 @@ def process_request(self, request):
         domain = get_object_or_404(get_tenant_domain_model().objects.select_related('tenant'),
                                    domain=hostname)
 
+        domain.tenant.domain_url = hostname
```

Comment:
I dont add the line 35, this line already existed, see here. :)
https://github.com/tomturner/django-tenants/blob/master/django_tenants/middleware.py